requirements.adoc

```diff
@@ -102,21 +146,27 @@ into schedulable/executable steps
 * one radiology study
 * one radiology report
 
-SPECIAL TERMS::
 
-*SCHEDULED:* A requested procedure is 'scheduled' if all of its
-scheduled procedures are 'scheduled'.
+```
+Should we delete this?
```

Comment:
we really need to agree on terms and their meaning. this doc will serve as a goto guide for new devs and is also the requirement. it is important to define/agree on what it means to be SCHEDULED/COMPLETED, ... in the context of an order, study, ...

this will lead to the business logic in the module, if we dont make this properly now the module will not reflect the real world and we'll just have wasted time and a module thats released and hard to change. this is really important!